src/containers/Form/index.js

```diff
@@ -24,6 +25,14 @@ export class Form extends Component {
     }
   }
 
+  static getDerivedStateFromProps(props, state) {
```

Comment:
When we get together next, please run through the use of static on the getDerivedStateFromProps.